ModelReplacementAPI/Libraries/.gitshit

```diff
@@ -0,0 +1 @@
+PUT THE DLL REQUIRED FOR THE PROJECT IN THIS FOLDER
```

Comment:
I can add a section that has links to each dll needed for building, but for licensing reasons I don't plan on including other mod dlls here. This is entirely for compatibility.

Reply:
This is only a placeholder to tell people to put the required dlls in there